src/OpenTelemetry.AutoInstrumentation.Native/bytecode_instrumentations.h

```diff
@@ -0,0 +1,13 @@
+// Auto-generated file, do not change it - generated by the IntegrationsJsonGenerator
+#ifndef BYTECODE_INSTRUMENTATIONS_H
+#define BYTECODE_INSTRUMENTATIONS_H
+
+#include "string.h"
+
+namespace trace
+{
+inline std::unordered_map<WSTRING, WSTRING> trace_integration_names = {{WStr("StrongNamedValidation"), WStr("OTEL_DOTNET_AUTO_TRACES_StrongNamedValidation_INSTRUMENTATION_DISABLED")}, {WStr("StackExchangeRedis"), WStr("OTEL_DOTNET_AUTO_TRACES_StackExchangeRedis_INSTRUMENTATION_DISABLED")}, {WStr("NServiceBus"), WStr("OTEL_DOTNET_AUTO_TRACES_NServiceBus_INSTRUMENTATION_DISABLED")}, {WStr("MySqlData"), WStr("OTEL_DOTNET_AUTO_TRACES_MySqlData_INSTRUMENTATION_DISABLED")}, {WStr("MongoDB"), WStr("OTEL_DOTNET_AUTO_TRACES_MongoDB_INSTRUMENTATION_DISABLED")}, {WStr("GraphQL"), WStr("OTEL_DOTNET_AUTO_TRACES_GraphQL_INSTRUMENTATION_DISABLED")}};
```

Comment:
Should we recommend `{0}` text to be uppercase in environmental variable? 
Most generic specification recommends uppercase, numbers and underscore.

Reply:
Issue already created. I do not want to change to many things in one PR #2094